### ISSUE TITLE
iOS SR not reading form validation alert

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1059]
"US03 Reading the What Is Health Equity > What Is Health Equity? page
1.8"	Screen Reader	N/A	After entering an invalid email address the warning is not announced to the screen reader user_mWeb(iOS)_1.8	"Environment:
DEVICE: iPad Air 2
iOS Version: 14.5.1
Model Number: MH0W2LL/A
Chrome browser: 87.0.4280.163

Pre-requisites: Screen reader is ON

Steps to reproduce:
1. Go to https://dev.healthequitytracker.org, Login with valid credentials
2. Navigate to the What Is Health Equity? page /whatishealthequity
3. Navigate to the ""Sign up for our newsletter"" field
4. Enter any invalid email address
5. Click submit

Actual Results:
After submitting an invalid email an error is displayed above the email field but not read by the screen reader.

Expected Results: 
All warnings should be read by the screen reader.

Video:
https://drive.google.com/file/d/1Yi2jGidanQgCRoWAZ7DUpVNmnkncVOcf/view?usp=sharing&resourcekey=0-H2v19Erd1SsQX3VZWRdFmA"

**Comment by benhammondmusic** [comment]
I was not able to replicate this error on desktop with NVDA-Firefox-Window nor Voiceover-Brave-Mac. This needs to be tested on iOS with screen reader enabled

**Comment by benhammondmusic** [comment]
closing as this is a browser limitation... currently voiceover reads the built in browser validation alert. we do not perform any client side validation, since this is really just a portal for the mailing list page that does the signup
